### PR TITLE
get all attributes

### PR DESCRIPTION
**Pull request by Achillx**
Extended attribute getter to retrieve all stored attributes of a face or faces.

### What type of change is this?
- [x] New feature in a **backwards-compatible** manner.

### Checklist

1. [x] Add the change to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading: `Added`, `Changed`, `Removed`.
1. [x] Run all tests on your computer (i.e. `invoke test`).

### CODE REVIEW

**Comment by Achillx** [comment]
Let me know if it's ok and I can implement it also for vertices and edges.

**Review by tomvanmele — COMMENTED** [review]
somehow it feels more natural to write `get_face_attributes_all` :)

File: src/compas/datastructures/_mixins/attributes.py [hunk 1]
```diff
@@ -768,6 +768,67 @@ def get_faces_attributes(self, keys, names, values=None):
...
+        * :meth:`get_face_attribute`
+        * :meth:`get_face_attributes`
+        * :meth:`get_faces_attribute`
+        * :meth:`get_faces_attributes`
+        * :meth:'get_faces_all_attributes'
+
+        """
+        if data:
+            return self.facedata[key]
+        else:
```
Comment (tomvanmele) [inline src/compas/datastructures/_mixins/attributes.py:800]: should this be an editable dict or a copy?

Comment (Achillx) [reply src/compas/datastructures/_mixins/attributes.py:800]: Whatever is best. It can also return a copy.

**Comment by Achillx** [comment]
> somehow it feels more natural to write `get_face_attributes_all` :)

to me, more natural was `get_all_face_attributes` but it messes the `get`-`what type`-`attribute` convention!
I'll change it to `get_face_attributes_all`

File: src/compas/datastructures/_mixins/attributes.py (hunk 1 above)
Comment (tomvanmele) [reply src/compas/datastructures/_mixins/attributes.py:800]: perhaps it could be an option?

**Comment by tomvanmele** [comment]
if you resolve the conflicts i will merge...

**Comment by tomvanmele** [comment]
conflicts?

**Pull request merged**